### ISSUE TITLE
Data overfitting and wrong classification result

### ISSUE THREAD

**Issue by vishalghor** [BartyzalRadek/Multi-label-Inception-net#9]
hi @BartyzalRadek ,

I followed the code changes which you did for multi label classification for a custom dataset.

But the model tends to  get overfit and also the output classification result for a images is not having individual probablities as you have depicted in the sample image i.e. one wiith car having 0.64 and accident as 0.39 probability.
i have tried your code for flower photos dataset similar to the dataset used in single label implementation by tensorflow.
but i am getting probability same as single label classifiation.

Kindly help me in resolving this issue.

**Comment by tianyuecao** [comment]
Hi, I have the similar question with you, I train on MS-COCO 2014, and find that the results only classify some of the pictures into 'person' but no other classes. Have you solved your problem? Tks.

**Comment by vishalghor** [comment]
Hi @tianyuecao i had resolved this issue, in my case the issue was due to not proper shuffling of data and the probabilty were resolved by using tf.sigmoid for activation.

**Comment by civilman628** [comment]
@tianyuecao  @tianyuecao  I have the same issue. i think the activation function is already sigmoid.  which line need to change?
